vars/kola.groovy

```diff
@@ -30,8 +30,8 @@ def call(params = [:]) {
     // Prevents multiple runs overwriting same filename in archiveArtifacts
     def token = shwrapCapture("uuidgen | cut -f1 -d-")
 
-    // Create a unique output directory for this run of fcosKola
-    def outputDir = shwrapCapture("mktemp -d ${cosaDir}/tmp/fcosKola-XXXXX")
+    // Create an unique output directory for this run of kola
```

Comment:
this should stay "a unique" right?

Reply:
since unique starts with a Y sound

Reply:
cc @jlebon since you made the suggestion. I have a feeling I'm about to learn something :)

Reply:
You can blame me @dustymabe =) fixing it